src/runtime/simulator.cc

```diff
@@ -537,7 +542,7 @@ CostMetrics Simulator::measure_operator_cost(Op const *op,
     ProfilingRecordKey key{params, mv};
     if (this->strict_hash_to_operator_cost.find(key) ==
         this->strict_hash_to_operator_cost.end()) {
-      CostMetrics cost_metrics;
+      CostMetrics cost_metrics{};
```

Comment:
Why not default initialization here?

Reply:
? Yes, this should have {}. And I think my code does have {}... but not sure why github PR doesn't have that. I'll check again once updating the PR to ensure it has {}.

Reply:
Oh, I see what you meant... Well, cost_metrics{} is to default initialize it.

Reply:
Yeah, just wanted to understand how it was different because it clearly was intentionally changed (it's the only line changed here :slightly_smiling_face:)

Reply:
Hmm, no much difference. It's just a good practice to always initialize a new variable. This was actually changed due to the incorrect sync cost of various parallel ops (solved previously). So, this `{}` may not be necessary. But it's a good practice.